hail/Makefile

```diff
@@ -272,19 +272,9 @@ $(eval $(call ENV_VAR,cloud_base))
 $(eval $(call ENV_VAR,wheel_cloud_path))
 
 
-.PHONY: generate-pip-lockfiles
-generate-pip-lockfiles:
-	HAIL_HAIL_DIR=$(HAIL_HAIL_DIR) \
-	../generate_pip_lockfile.sh python/hailtop/requirements.txt python/hailtop/pinned-requirements.txt && \
-	HAIL_HAIL_DIR=$(HAIL_HAIL_DIR) \
-	../generate_pip_lockfile.sh python/requirements.txt python/pinned-requirements.txt && \
-	HAIL_HAIL_DIR=$(HAIL_HAIL_DIR) \
-	../generate_pip_lockfile.sh python/dev/requirements.txt python/dev/pinned-requirements.txt
-
-
 .PHONY: check-pip-lockfile
 check-pip-lockfile:
-	$(PIP) install pip-tools && bash ../check_pip_requirements.sh python/requirements.txt python/pinned-requirements.txt
+	$(PIP) install pip-tools && bash ../check_pip_requirements.sh python
```

Comment:
So, `pytest` depends on INIT_SCRIPTS which depends on this which means I have to invoke `pip` to do a no-op install and then invoke pip-compile. Not great. It's been bothering me lately when I'm doing iterative development using the tests. Can we check the pip lock file once and be done with it?

Reply:
This has been bothering me too. Yes please.

Reply:
We primarily added this `check-pip-lockfile` target to make sure that someone running `install-on-cluster` doesn't add some requirements and then get surprised when they're not installed because they didn't change the lockfiles. This is already a dependency of `install-on-cluster` so I just removed it as a dependency of the `deploy.yaml`.